### PR TITLE
Add Ukrainian holidays

### PR DESCRIPTION
**Pull request by kapi14**
Hi Spatie! 

There is one extra function for calculating the correct Eastern date due to the Meeus Julian algorithm or Revised Julian calendar

### CODE REVIEW

**Comment by Nielsvanpach** [comment]
Could you look at the failing tests? Thanks!

**Comment by kapi14** [comment]
Hi @Nielsvanpach,

Sure, should be good now.

File: src/Countries/Ukraine.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            ->setTimezone('Europe/Kyiv');
+
+        return [
+            'Великодній Понеділок' => $easter->addDay(),
+            'Трійця' => $easter->addDays(50),
+        ];
+    }
+
+
+    protected function orthodox_eastern($year): string
```
Comment (Nielsvanpach) [inline src/Countries/Ukraine.php:47]: In Country.php we already have a orthodox calculation. Could you verify if that works?

File: src/Countries/Ukraine.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            'День Незалежності України' => '08-24',
+            'День захисників і захисниць України' => '10-01',
+            'Різдво Христове' => '12-25',
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+        $easter = CarbonImmutable::createFromTimestamp($this->orthodox_eastern($year))
```
Comment (Nielsvanpach) [inline src/Countries/Ukraine.php:37]: You can use $this->easter($year) instead

File: src/Countries/Ukraine.php [hunk 3]
```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Ukraine extends Country
+{
+    protected function __construct(
```
Comment (Nielsvanpach) [inline src/Countries/Ukraine.php:9]: If you don't need the constructor, you can remove it

**Review by Nielsvanpach — COMMENTED** [review]
Thanks!

File: src/Countries/Ukraine.php (hunk 1 above)
Comment (kapi14) [reply src/Countries/Ukraine.php]: Perfect, yes this is working correctly!

**Comment by Nielsvanpach** [comment]
PR looks good! 👍  I'll leave this one open until I've seen some approvals!

**Comment by kapi14** [comment]
Thanks for reviewing.

**Review by esurov — APPROVED** [review]
lgtm

**Comment by Nielsvanpach** [comment]
Thanks!

**Pull request merged**